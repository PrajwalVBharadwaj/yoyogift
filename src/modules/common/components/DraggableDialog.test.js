import React from "react";
import DraggableDialog from "./DraggableDialog";
import { shallow, mount } from "../../../Enzyme";

describe("Testing DraggableDialog", () => {
  it("Snapshot testing", () => {
    const wrapper = shallow(<DraggableDialog />);
    expect(wrapper).toMatchSnapshot();
  });
});
