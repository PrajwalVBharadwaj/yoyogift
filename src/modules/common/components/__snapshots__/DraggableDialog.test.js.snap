// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing DraggableDialog Snapshot testing 1`] = `ShallowWrapper {}`;

exports[`Testing DraggableDialog Snapshot testing 2`] = `ReactWrapper {}`;
